service/controller/chart/resource/status/create.go

```diff
@@ -80,7 +80,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		AppVersion: releaseContent.AppVersion,
 		Reason:     reason,
 		Release: v1alpha1.ChartStatusRelease{
-			LastDeployed: v1alpha1.DeepCopyTime{Time: releaseContent.LastDeployed},
+			LastDeployed: metav1.Time{Time: releaseContent.LastDeployed},
```

Comment:
After `apiextension` update, we are going to use different packages for this deep copying.